fossunited/www/book-conference-ticket/index.html

```diff
@@ -134,7 +134,7 @@ <h3>Book Conference Ticket for {{ event.event_name }}</h3>
           <select
             id="state"
             name="State"
-            class="form-control custom-select"
+            class="form-control custom-select form-select"
```

Comment:
did you test the form-select? Make sure the layout is not breaking

Reply:
Yes, I have checked. and it works without breaking